file-browser.lua

```diff
@@ -1121,7 +1145,9 @@ local function loadfile(item, flag, autoload, directory)
         return loadlist(path, flag) end
 
     if sub_extensions[ API.get_extension(item.name, "") ] then
-        mp.commandv("sub-add", path, flag == "replace" and "select" or "auto")
+        mp.commandv("sub-add", path, flag == "replace" and "cached" or "select" or "auto")
+    elseif audio_extensions[ API.get_extension(item.name, "") ] then
+        mp.commandv("audio-add", path, flag == "replace" and "auto" or "cached" or "select")
```

Comment:
```suggestion
        mp.commandv("sub-add", path, flag == "replace" and "select" or "auto")
    elseif audio_extensions[ API.get_extension(item.name, "") ] then
        mp.commandv("audio-add", path, flag == "replace" and "select" or "auto")
```

For the time being I'd like to stick to the original behaviour. I like the idea of using the cached option, but I'd like to put it behind another script-opt, I'd also like for it to be done in a different commit/PR (I might do it myself once this is merged). If certain versions of mpv or certain playback options cause a freeze when using the audio-add command, then that's not our fault, I don't want to make the behaviour of this script inconsistent because of a bug that could be fixed at any time.

What you had here was bugged anyway, since the `auto` option for `sub-add` and the `select` option for `audio-add` would never be triggered due to Lua's ternary statement behaviour.

Reply:
I agree with you that this is a probably issue of mpv or ffmpeg, and should be reported and waiting for their repair. 
Let's return to original behavior.